03-Custom-Actions-In-Python/03-Custom-Action-Features.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>This will be a big abstract to users without a concrete example. Is this really necessary at the tutorial level or could users find it in the reference docs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>My concern there is someone will look at the tutorial expect the pressure, and we get an email stating that they want support for having pressure for an action. I wouldn't expect people to read the reference documentation.</p>

Reply:
If we don't expect people to read the reference documentation, then why do we even write it?

Reply:
<p>Fair enough.</p>